arm/cookbooks/motd/README.md

```diff
@@ -81,3 +81,36 @@ kitchen test
 This takes:
   t2.medium: 6m02s, 5m50s
   m3.large: 5m37s, 5m53s
+
+
+## Set up with kitchen-azurerm
+
+https://github.com/pendrica/kitchen-azurerm
+
+Assumes you have already set up an Azure account.
+
+0. Install gem(s) `/azure-sandbox$ bundle`
+0. Install and configure cli `brew install azure-cli && azure login`
+0. `cp .kitchen.local.azure.example.yml .kitchen.local.yml` and customize
+0. Follow along here to create a service principal with Contributor permissions: https://github.com/pendrica/kitchen-azurerm#configuration.  For me it looked like:
+```
+azure ad app create -n kitchen --home-page http://www.contoso.org --identifier-uris https://www.contoso.org/example -p <password>
+azure ad sp create -a <AppId from last output>
+azure role assignment create --objectId <Object Id from last output> -o Contributor  -c /subscriptions/<subscription_id>/
+```
+0. Configure Kitchen/azurerm. Easiest is to export some ENV vars. Another option is to use a `~/.azure/config` file, more info see kitchen-azurerm docs.
+```
+export AZURE_CLIENT_ID="<AppId from above>"
+export AZURE_CLIENT_SECRET="<Password from above>"
+export AZURE_TENANT_ID="<from `azure account show`>"
+export SUBSCRIPTION_ID="<from `azure account list`>"
+```
+
+0. `kitchen test`.  If this fails trying to delete non-existing resources, `rm -rf .kitchen` and try again.
+
+Benchmarks:
+
+| VM | Cores | Memory | t1 | t2 | t3 |
+| -- | -- | -- | -- | -- | -- |
+| Standard_D1 | 1 | 3.5G | 9:00 | 9:15 | 7:35 |
+| Standard_D2 | 2 | 7G | 8:01 | 9:53 | 8:04 |
```

Comment:
I was keeping some notes on long these tests take to run. After the zippiness of running kitchen w/ Docker the 8min wait for win2012r2 to spin up, converge, validate, and destroy itself feels interminable. Whether there's any remedy for that is unclear. How about change line 111ff to:

```
Benchmarks:
 
Full integration tests against Win2012R2 take a _long_ time. Here's what we observed in minutes:seconds for various instance types:

| VM | Cores | Memory | test1 mm:ss | test2 mm:ss | test3 mm:ss |
| -- | -- | -- | -- | -- | -- |
| AzureSM Standard_D1 | 1 | 3.5G | 9:00 | 9:15 | 7:35 |
| AzureSM Standard_D2 | 2 | 7G | 8:01 | 9:53 | 8:04 |
```